semgrep_output_v1.atd

```diff
@@ -102,6 +102,13 @@ type rule_id
 (*****************************************************************************)
 (* EXPERIMENTAL: Do not rely on those internal types, they might disappear *)
 
+type engine_kind
+  <ocaml attr="deriving show">
+  <python decorator="dataclass(frozen=True)"> = [
+  | OSSMatch
```

Comment:
if the type is called engine_kind, I would not add the Match suffix then
but OssEngine or ProEngine.